core/src/main/resources/data/upgrade_5.3.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE software ALTER COLUMN versionEndExcluding SET DATA TYPE VARCHAR(75);
```

Comment:
as @Keymaster65 already suggested, i would also prefer to take the column length of 255 chars:

```suggestion
ALTER TABLE software ALTER COLUMN versionEndExcluding SET DATA TYPE VARCHAR(255);
ALTER TABLE software ALTER COLUMN versionEndIncluding SET DATA TYPE VARCHAR(255);
ALTER TABLE software ALTER COLUMN versionStartExcluding SET DATA TYPE VARCHAR(255);
ALTER TABLE software ALTER COLUMN versionStartIncluding SET DATA TYPE VARCHAR(255);
```